lib/kafo/configuration.rb

```diff
@@ -240,6 +240,23 @@ def answers
       @data
     end
 
+    def run_migrations
+      migrations = Kafo::Migrations.new(migrations_dir)
+      migrations.load_migrations
+      @app, @data = migrations.run(app, answers)
+      if migrations.processed > 0
+        @modules = nil
```

Comment:
I guess this is to reload modules, could you add short comment here if it's the case?